acceptance/Rakefile

```diff
@@ -9,9 +9,13 @@ ONE_DAY_IN_SECS = 24 * 60 * 60
 REPO_CONFIGS_DIR = "repo-configs"
 CLEAN.include('*.tar', REPO_CONFIGS_DIR, 'merged_options.rb')
 
+# If elsewhere we're depending on internal network resources
+# then assume we can depend on them here
```

Comment:
This comment dates to the previous implementation but doesn't make sense anymore.